pages/common/git-ls-tree.md

```diff
@@ -0,0 +1,12 @@
+# git ls-tree
+
+> List the contents of a tree object.
+> More information: <https://git-scm.com/docs/git-ls-tree>.
+
+- List the contents of the tree on a branch:
+
+`git ls-tree {{master}}`
```

Comment:
I'd have a slight preference to use `{{branch_name}}` instead of `{{master}}` here, but I'm open to being convinced otherwise.

Reply:
Fixed in 35035345650e07534a454774f0aa987ca60c6124